cmake/onednn_lite_config.cmake

```diff
@@ -0,0 +1,353 @@
+include_guard()
+
+get_property(DNNL_INCLUDES GLOBAL PROPERTY GC_DNNL_INCLUDES)
+get_property(DNNL_PATH GLOBAL PROPERTY GC_DNNL_SOURCE_DIR)
+if (NOT DEFINED DNNL_INCLUDES)
+    return()
```

Comment:
This looks like a config error to me. Can this report an error msg?

Reply:
Changed to error msg & halt.